src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -93,5 +97,25 @@ internal override void Close()
                 this.ViewModel.Dispose();
             }
         }
+
+        private void Generate_Editorconfig(object sender, System.Windows.RoutedEventArgs e)
+        {
+            var optionSet = this.ViewModel.ApplyChangedOptions(this.OptionService.GetOptions());
+            var editorconfig = _getCurrentEditorConfigOptionsString(optionSet);
+
+            var sfd = new System.Windows.Forms.SaveFileDialog
+            {
+                Filter = "All files (*.*)|",
+                FileName = ".editorconfig",
+                Title = "Save .editorconfig File"
```

Comment:
1. this title likely needs to be localized.
2. 'File' should be be lowercased i believe.

Reply:
Thanks! Just submitted a [request](https://github.com/dotnet/roslyn/issues/28584) to have this localized.